src/features/recipe/components/add/RecipeIngredientAddListItem.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from "react";
+import { StyleSheet, View, ViewProps } from "react-native";
+
+import { Text } from "@/components/Text";
+import { Ingredient } from "@/models";
+
+export type RecipeIngredientAddListItemProps = ViewProps & {
+  item: Ingredient;
+};
+
+export const RecipeIngredientAddListItem: FC<
+  RecipeIngredientAddListItemProps
+> = ({ style, item, ...props }) => {
+  return (
+    <View {...props} style={[style, styles.container]}>
```

Comment:
Maybe you could've used one text?
Another option is to wrap these three texts with another `Text` setting that one to `xs`. In that case you most likely won't need the wrapping view